pkg/queue/request_metric.go

```diff
@@ -211,8 +205,15 @@ func (h *appRequestMetricsHandler) ServeHTTP(w http.ResponseWriter, r *http.Requ
 	h.next.ServeHTTP(rr, r)
 }
 
+/*
+const (
+	defaultTagName   = "DEFAULT"
+	undefinedTagName = "UNDEFINED"
+	disabledTagName  = "DISABLED"
+)
+
 // GetRouteTagNameFromRequest extracts the value of the tag header from http.Request
-/*func GetRouteTagNameFromRequest(r *http.Request) string {
```

Comment:
I know you didn't comment this method out in the first place, but while we're here wdyt about adding a link to https://github.com/knative/serving/pull/8969 to explain why this is commented in the first place and when it will be uncommented?